docs/pages/docs/best-practices.mdx

```diff
@@ -1,39 +1,44 @@
 ---
 title: Docs Best Practices
+description: Best practices for documentation
 ---
 
-## Content
+# Documentation Best Practices
 
-**Keep it simple**
+This article serves to define [documentation](../docs) best practices.
 
-Read the book [On Writing Well](https://www.amazon.com/Writing-Well-Classic-Guide-Nonfiction/dp/0060891548).
+## Work Flow
 
-**Write for the target audience**
+This section describes the general workflow for writing documentation:
 
-Do not overload getting started guides with content and complex terminology, focus
-on benefits to the user. Do not add architecture diagrams in getting started guides.
-Likewise, do not do demos in architecture articles.
+1. Select a branch from [github.com/teleport](https://github.com/gravitational/teleport/) corresponding to the version of the documentation/product you'd like to work on.
```

Comment:
Nit: I'd just write `github.com/gravitational/teleport` since it's the actual repo path.